src/pages/Favoritos/index.js

```diff
@@ -1,52 +1,54 @@
-import './index.css'
-import { useEffect, useState} from 'react'
+// Packages
+import React, { useEffect, useState} from 'react'
 import { Link } from 'react-router-dom'
 import { toast } from 'react-toastify'
 
-function Favoritos() {
+// Styles
+import './index.css'
 
-    const [filmes, setFilmes] = useState([])
+function Favoritos() {
+    const [movies, setMovies] = useState([])
 
     useEffect(() => {
+    const myList = localStorage.getItem('filmes')
 
-    const minhaLista = localStorage.getItem('filmes')
-    setFilmes(JSON.parse(minhaLista) || [])
-
+    setMovies(JSON.parse(myList) || [])
     },[])
```

Comment:
Esqueci de adicionar espaco aqui.
Sugestão: adicionar regras de lint (eslint)

```suggestion
    }, [])
```